# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../djbdns/head_n.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2004-11-23 00:29:39.000000000 -0300
+++ ./Makefile	2004-11-23 00:31:09.000000000 -0300
@@ -31,7 +31,7 @@
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -1 conf-home` > auto_home.c
+	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
@@ -205,14 +205,14 @@
 choose: \
 warn-auto.sh choose.sh conf-home
 	cat warn-auto.sh choose.sh \
-	| sed s}HOME}"`head -1 conf-home`"}g \
+	| sed s}HOME}"`head -n 1 conf-home`"}g \
 	> choose
 	chmod 755 choose
 
 compile: \
 warn-auto.sh conf-cc
 	( cat warn-auto.sh; \
-	echo exec "`head -1 conf-cc`" '-c $${1+"$$@"}' \
+	echo exec "`head -n 1 conf-cc`" '-c $${1+"$$@"}' \
 	) > compile
 	chmod 755 compile
 
@@ -445,7 +445,7 @@
 dnstracesort: \
 warn-auto.sh dnstracesort.sh conf-home
 	cat warn-auto.sh dnstracesort.sh \
-	| sed s}HOME}"`head -1 conf-home`"}g \
+	| sed s}HOME}"`head -n 1 conf-home`"}g \
 	> dnstracesort
 	chmod 755 dnstracesort
 
@@ -566,7 +566,7 @@
 warn-auto.sh conf-ld
 	( cat warn-auto.sh; \
 	echo 'main="$$1"; shift'; \
-	echo exec "`head -1 conf-ld`" \
+	echo exec "`head -n 1 conf-ld`" \
 	'-o "$$main" "$$main".o $${1+"$$@"}' \
 	) > load
 	chmod 755 load
@@ -754,7 +754,7 @@
 rts: \
 warn-auto.sh rts.sh conf-home
 	cat warn-auto.sh rts.sh \
-	| sed s}HOME}"`head -1 conf-home`"}g \
+	| sed s}HOME}"`head -n 1 conf-home`"}g \
 	> rts
 	chmod 755 rts
 
@@ -897,8 +897,8 @@
 systype: \
 find-systype.sh conf-cc conf-ld trycpp.c x86cpuid.c
 	( cat warn-auto.sh; \
-	echo CC=\'`head -1 conf-cc`\'; \
-	echo LD=\'`head -1 conf-ld`\'; \
+	echo CC=\'`head -n 1 conf-cc`\'; \
+	echo LD=\'`head -n 1 conf-ld`\'; \
 	cat find-systype.sh; \
 	) | sh > systype
 
